TLM/TLM/Resources/lang_pl.txt

```diff
@@ -231,4 +231,13 @@ Also_apply_to_left/right_turns_between_one-way_streets Uwzględnia również skr
 Scan_for_known_incompatible_mods_on_startup Uruchamiaj sprawdzanie przy starcie gry
 Ignore_disabled_mods Ignoruj wyłączone mody
 Traffic_Manager_detected_incompatible_mods Traffic Manager wykrył niekompatybilne mody
-Notify_me_if_there_is_an_unexpected_mod_conflict Powiadom mnie w razie nieoczekiwanej niezgodność modów
\ No newline at end of file
+Notify_me_if_there_is_an_unexpected_mod_conflict Powiadom mnie w razie nieoczekiwanej niezgodność modów
+Keybind_toggle_TMPE_main_menu Toggle TM:PE Main Menu
+Keybind_toggle_traffic_lights_tool Use 'Toggle Traffic Lights' Tool
+Keybind_use_lane_arrow_tool Use 'Lane Arrow' Tool
+Keybind_use_lane_connections_tool Use 'Lane Connections' Tool
+Keybind_use_priority_signs_tool Use 'Priority Signs' Tool
+Keybind_use_junction_restrictions_tool Use 'Junction Restrictions' Tool
+Keybind_use_speed_limits_tool Use 'Speed Limits' Tool
+Keybind_lane_connector_stay_in_lane Lane connector: Stay in lane
+Keybinds Keybinds
```

Comment:
You can replace those translation keys 😉 

Keybind_toggle_TMPE_main_menu `Menu TM:PE`
Keybind_toggle_traffic_lights_tool `Narzędzie 'Przełącznik Sygnalizacji świetlnej' `
Keybind_use_lane_arrow_tool `Narzędzie 'Strzałki pasów ruchu'`
Keybind_use_lane_connections_tool `Narzędzie 'Połącz pasy ruchu'`
Keybind_use_priority_signs_tool `Narzędzie 'Znaki pierwszeństwa'`
Keybind_use_junction_restrictions_tool `Narzędzie 'Ograniczenia na skrzyżowaniu'`
Keybind_use_speed_limits_tool `Narzędzie 'Limity prędkości'`
Keybind_lane_connector_stay_in_lane `Połącz pasy ruchu: Pozostań na pasie`
Keybinds `Skróty klawiszowe`

Reply:
Added

Reply:
https://github.com/krzychu124/Cities-Skylines-Traffic-Manager-President-Edition/blob/d68efff147db6802173f08e1c4afb8d4195b7a36/TLM/TLM/Resources/lang_pl.txt#L66

Please remove ` (dodatkowe informacje) ` from that translation key to be able to see keybinds tab while Polish lang selected 😉

Reply:
Just tested Polish locale and seems this change was already made.